lms/djangoapps/django_comment_client/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls.defaults import url, patterns, include
 
 urlpatterns = patterns('',  # nopep8
-    url(r'forum/?', include('django_comment_client.forum.urls')),
-    url(r'', include('django_comment_client.base.urls')),
+                       url(r'forum/', include('django_comment_client.forum.urls')),
```

Comment:
hrm, can I suggest we change this indenting to be:

urlpatters = patterns(
    '',
    url(..)
    :
)